### DISCUSSION TITLE
VRT in Typescript

### DISCUSSION THREAD

**Discussion by Andyruu** [Visual-Regression-Tracker/Visual-Regression-Tracker#441] [Q&A]
Hi,

I'm curious if anyone has been able to get VRT working with Typescript?

**Comment by pashidlos** [comment]
@Andyruu thanks for the question 
sure, you could use it with typescript 
just add types in `tsconfig.json`
something like 
```
    "types": [
      "cypress",
      "@visual-regression-tracker/agent-cypress/dist"
    ]
```

**Reply by Andyruu** [reply]
Thanks for the quick response! That seems to work, the compiler still complains the property doesn't exist for the type, but the commands run successfully so 🤷‍♂️. Likely something to do with adding the commands to the cypress agent
